R-random-forest/random_forest_SigOpt.R

```diff
@@ -0,0 +1,82 @@
+# install packages
+install.packages("devtools", repos = "http://cran.us.r-project.org")
+library(devtools)
+install_github("sigopt/SigOptR")
+library(SigOptR)
+install.packages('randomForest', repos = "http://cran.us.r-project.org")
+library(randomForest)
+
+# set SigOpt client token
+# get your client API token here : https://sigopt.com/user/profile
```

Comment:
I use this line in other examples:
`# Learn more about authenticating the SigOpt API: sigopt.com/docs/overview/authentication`